src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoGetAbbreviatedDayName.cs

```diff
@@ -16,8 +16,7 @@ public static IEnumerable<object[]> GetAbbreviatedDayName_TestData()
             yield return new object[] { new CultureInfo("en-US").DateTimeFormat, englishAbbreviatedDayNames };
             yield return new object[] { new DateTimeFormatInfo(), englishAbbreviatedDayNames };
 
-            // ActiveIssue(2103)
-            if (!PlatformDetection.IsUbuntu1510 && !PlatformDetection.IsUbuntu1604 && !PlatformDetection.IsUbuntu1610)
+            if (!PlatformDetection.IsUbuntu || PlatformDetection.IsUbuntu1404)
```

Comment:
My point was just that this is an extra test case we're adding for 14.04, and I'm not sure we need to being going out of our way to add such test cases for older versions when they then don't work on subsequent versions.  We could simply drop the additional test.  But I'm fine leaving it for now.